src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/clients/IGDBClientTests.kt

```diff
@@ -1,4 +1,169 @@
 package com.gamingbacklog.api.gamingbacklogapi.clients
 
+import com.gamingbacklog.api.gamingbacklogapi.testutil.models.MockResponse
```

Comment:
How do we want to organize/name our tests to differentiate unit and integration tests? Since we're mocking the response below I assume this is a unit test - are we also going to want integration tests for our IGDB client?

Reply:
I'm not sure how much integration we'll need for IGDBClient specifically. Like maybe 1-2 but for integration we may just want to test the controllers that end up calling it. I will create a separate unit directory to differentiate the two test types for now though.

Reply:
I guess what I was thinking of is more along the lines of third party/end-to-end tests. Don't need to worry about that now, but good to think of down the line